src/routes/index.tsx

```diff
@@ -96,6 +97,7 @@ const getRoutesList = () => {
         path="/profile/:gtfsRideGtfsRouteId"
         key={'/profile/:gtfsRideGtfsRouteId'}
         element={<Profile />}
+        ErrorBoundary={ErrorPage}
```

Comment:
Hmm, is it the only route that we should add ErrorBoundary?
Not looks like it, I changed the VITE_BASE_PATH to be an uncorrect path and still I don't see any data and error in the console...
![image](https://github.com/hasadna/open-bus-map-search/assets/76536506/713b8f99-5bf7-4c75-9dd8-5aa2dc737e31)
![image](https://github.com/hasadna/open-bus-map-search/assets/76536506/5714337f-87b6-46be-b4ca-93a00b716fb5)

Reply:
All routes should have errorboundaries. It can be done on another PR, Im not sure what changes you require

Reply:
I meant even when the backend is off , this error page is not displayed. From first look of it, we don't catch errors from axios requests. Up to you, if you would like to do it as part of this pr or we can clarify it in the issue that this is left to do.

Reply:
That's a great idea actually.
This error boundary build for handling js exceptions rather than network errors, but it's definitely something we should consider